easybuild/cray_external_modules_metadata.cfg

```diff
@@ -0,0 +1,69 @@
+[cray-netcdf/4.3.2]
+name = netCDF,netCDF-Fortran
+version = 4.3.2, 4.3.2
+prefix = NETCDF_DIR
+
+[cray-hdf5/1.8.13]
+name = HDF5
+version = 1.8.13
+prefix = HDF5_DIR
+
+[cray-hdf5-parallel/1.8.13]
+name = HDF5
+version = 1.8.13
+prefix = HDF5_DIR
+
+[cray-libsci/13.0.4]
+name = LIBSCI
+version = 13.0.4
+prefix = CRAY_LIBSCI_PREFIX_DIR
+
+[cray-libsci/13.2.0]
+name = LIBSCI
+version = 13.2.0
+prefix = CRAY_LIBSCI_PREFIX_DIR
+
+[cudatoolkit/6.5.14-1.0502.9613.6.1]
+name = CUDA
+version = 6.5.14-1.0502.9613.6.1,
+prefix = CRAY_CUDATOOLKIT_DIR
+
+[cudatoolkit/6.5.14-1.0502.9836.8.1]
+name = CUDA
+version = 6.5.14-1.0502.9836.8.1,
+prefix = CRAY_CUDATOOLKIT_DIR
+
+[cudatoolkit/7.0.28-1.0502.10742.5.1]
+name = CUDA
+version = 7.0.28-1.0502.10742.5.1
+prefix = CRAY_CUDATOOLKIT_DIR
+
+[cray-petsc-complex-64/3.5.3.1]
+name = PETSc
+version = 3.5.3.1
+prefix = PETSC_DIR
+
+[cray-petsc-complex/3.5.3.1]
+name = PETSc
+version = 3.5.3.1
+prefix = PETSC_DIR
+
+[cray-petsc/3.5.3.1]
+name = PETSc
+version = 3.5.3.1
+prefix = PETSC_DIR
+
+[cray-petsc-64/3.5.3.1]
+name = PETSc
+version = 3.5.3.1
+prefix = PETSC_DIR
+
+[fftw/3.3.4.5]
+name = FFTW
+version = 3.3.4.5
+prefix = FFTW_INC
+
+[fftw/3.3.4.3]
+name = FFTW
+version = 3.3.4.3
+prefix = FFTW_INC
```

Comment:
please change to `FFTW_INC/..`, cfr. https://github.com/hpcugent/easybuild-framework/pull/1637